docs/usage.md

```diff
@@ -12,49 +12,85 @@ The input data can be passed to nf-core/mag in two possible ways, either using t
 
 You can specify a CSV samplesheet input file that contains the paths to your FASTQ files and additional metadata. Furthermore when a `run` column is present, the pipeline will also run perform run- or lane-wise concatenation, for cases where you may have a sample or library sequenced with the same sequencing configuration across multiple runs. The optional run merging happens after short read QC (adapter clipping, host/PhiX removal etc.), and prior to normalisation, taxonomic profiling, and assembly.
 
+If short reads are provided (`short_reads_1`), the `short_reads_platform` column is required. Valid options include:
+
+- `ILLUMINA`, `BGISEQ`, `LS454`, `ION_TORRENT`, `DNBSEQ`, `ELEMENT`, `ULTIMA`, `VELA_DIAGNOSTICS`, `GENAPSYS`, `GENEMIND`, `TAPESTRI`.
+
+If long reads are provided (`long_reads`), the `long_reads_platform` column is required. Valid options include:
+
+- `OXFORD_NANOPORE`: Oxford Nanopore Technologies (ONT) reads, which may have higher error rates compared to newer technologies.
+- `OXFORD_NANOPORE_HQ`: High-quality ONT reads, typically with an error rate of less than 5%, achievable with the latest ONT chemistries and sequencing platforms. This option should generally be used unless working with older data.
+- `PACBIO_SMRT`: Pacific Biosciences Single Molecule Real-Time (SMRT) sequencing reads.
+
+These platform fields are important for downstream alignment and assembly tools.
+
 At a minimum CSV file should contain the following columns:
 
-`sample,group,short_reads_1,short_reads_2,long_reads`
+`sample,group,short_reads_1,short_reads_2,long_reads,short_reads_platform,long_reads_platform`
 
-The path to `long_reads` and `short_reads_2` is optional. Valid examples could look like the following:
+In cases of short-read only data run, the path to `long_reads` and `short_reads_2` is optional. Valid examples could look like the following:
```

Comment:
I find the phrasing a bit confusing. I interpret "At a minimum ... should" as required, but most fields are now optional since one can do short reads only, long reads only as well as in combination, right? Would a better title be "A typical CSV file for combined short ad long read assembly would contain the following columns:"? Plus some more details about what can be removed for the three different cases.